src/Athena/Views/Home/Home.cshtml

```diff
@@ -47,14 +47,50 @@ else
 
 }
 
+@Html.Partial("_CurrentScheduleModal", false)
+@Html.Partial("_UnmetDependencyError")
+
+<div id="modal2" class="modal">
+    <div class="modal-content">
+        <h4>Confirm Removal</h4>
+        <p>Please confirm that you would like to remove this class from your schedule.</p>
+    </div>
+    <div class="modal-footer">
+        <a href="#!" class="modal-action modal-close waves-effect btn" id="Remove-Course-Button">Remove</a>
+        <a href="#!" class="modal-action modal-close waves-effect btn">Cancel</a>
+    </div>
+</div>
+
 @section scripts {
     @if (User.Identity.IsAuthenticated)
     {
         <script type="text/javascript" src="@Url.Content("~/schedule.js")"></script>
         <script type="text/javascript">
+            function loadOtherPage(link) {
+                $("<iframe class='printpage'>") // create a new iframe element
+                    .attr("src", link) // point the iframe to the page link you want to print
+                    .appendTo("body");
+            }
+            function ConfirmRemove(event) {
```

Comment:
This function isn't needed as far as I can tell.

Reply:
its not